### ISSUE TITLE
Install imagick under MacOS

### ISSUE THREAD

**Issue by ruudboon** [shivammathur/setup-php#155]
Could you add support for imagick under MacOS?
See https://github.com/phalcon/cphalcon/pull/14729/checks?check_run_id=395790341#step:3:20

**Comment by shivammathur** [comment]
@ruudboon 
Will add it in `1.7.4`
Until add this step before the `setup-php` step
```yml
 - name: Install requirements for ext-imagick
   run: brew install pkg-config imagemagick
```

**Comment by ruudboon** [comment]
@shivammathur Awesome! Thnx again for your fast reply

**Comment by shivammathur** [comment]
@ruudboon 
Fixed in `1.7.4`
Now `ext-imagick` should install seamlessly.
